lua/plenary/path.lua

```diff
@@ -400,6 +404,16 @@ end
 function Path:close()
 end
 
+function Path:write(txt, flag, mode)
+  assert(flag, [[Path:write_text requires a flag! For example: 'w' or 'a']])
+
+  mode = mode or 438
```

Comment:
What if for some reason the user provided a bool true? would we still want to default to 438?

Reply:
If someone just passes in a boolean i want that his home_dir gets deleted :rofl:

Reply:
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

Reply:
We use that pattern for mode everywhere so i don't care :)